internal/dot/graph_test.go

```diff
@@ -36,7 +36,10 @@ func TestNewGraph(t *testing.T) {
 
 	assert.Equal(t, make(map[groupKey]*Group), g.groupMap)
 	assert.Equal(t, make(map[CtorID]*Ctor), g.ctorMap)
-	assert.Equal(t, &FailedNodes{}, g.Failed)
+	assert.Equal(t, &FailedNodes{
+		FailedCtors:  make(map[CtorID]struct{}),
+		FailedGroups: make(map[groupKey]struct{}),
+	}, g.Failed)
```

Comment:
not necessary. internal state as far as this code is concerned.

Reply:
in fact, this whole test is unnecessary. this is equivalent to writing the same code again except from the other side.

Reply:
It make sense that testing constructors are not as valuable. Removed.